Sources/SwiftPhoenixClient/Socket.swift

```diff
@@ -678,12 +708,14 @@ public class Socket: PhoenixTransportDelegate {
 
     }
 
+    urlComponents.queryItems = [URLQueryItem(name: "vsn", value: version.rawValue)]
+
     // If there are parameters, append them to the URL
-    if let params = params?() {
-      urlComponents.queryItems = params.map {
-        URLQueryItem(name: $0.key, value: String(describing: $0.value))
+      if let params = params?() {
+        urlComponents.queryItems?.append(contentsOf: params.map {
+          URLQueryItem(name: $0.key, value: String(describing: $0.value))
+        })
```

Comment:
indentation here doesn't match surrounding lines

Reply:
fixed in https://github.com/davidstump/SwiftPhoenixClient/pull/204/commits/a2fe76592cf627e988e2ccae80169b06cc6a8a22